CHANGELOG.md

```diff
@@ -84,6 +85,7 @@
 - PR #1307: Add RF regression benchmark
 - PR #1327: Update the code to build treelite with protobuf
 - PR #1289: Add Python benchmarking support for FIL
+- PR #1370: Add LabelBinarizer and MNMG LabelBinarizer
```

Comment:
I think this should be removed right?

Reply:
Yeah, I think we can remove it. I kept it originally to maintain a reference back to the original PR, but then the code was updated in this PR.